apps/extension/src/hooks/chain-id.ts

```diff
@@ -11,8 +26,8 @@ export const getChainId = async (): Promise<string> => {
 export const useChainIdQuery = () => {
   const { data, refetch } = useQuery({
     queryKey: ['chain-id'],
-    queryFn: getChainId,
-    refetchInterval: false,
```

Comment:
Given chain ids don't change on refetch, the `refetchInterval` was intended to instruct react-query not to refetch. But that was a misreading of the api, `staleTime: Infinity` is the correct one.